kvrocks.conf

```diff
@@ -61,6 +61,11 @@ cluster-enabled no
 # Default: no
 repl-namespace-enabled no
 
+# By default, the max length of bulk string is limited to 512MB. If you want to
+# change this limit to a different value(must >= 1MiB), you can use the following configuration.
+#
+# proto-max-bulk-len 536870912
```

Comment:
Should it be in MB or KB?

Reply:
Yes, I had thought about this. However the configuration's name didn't indicate the unit, so I think it will be better to use bytes.

Reply:
You can use `ParseSizeAndUnit` [here](https://github.com/apache/kvrocks/blob/unstable/src/common/parse_util.cc#L35), or you can rename it to proto-max-bulk-len-mb.

Reply:
Currently, the config rewrite will change it to the number of bytes. I have created an issue to support this: https://github.com/apache/kvrocks/issues/2445.